src/Components/Application.cpp

```diff
@@ -0,0 +1,48 @@
+#include "Application.h"
+#include <iostream>
+#include <string>
+
+#include <opencv2/opencv.hpp>
+
+using namespace std;
+Controller* Application::controller = nullptr;
+
+Application::Application()
+{
+	controller = new Controller();
+	controller->init(nullptr);
+}
+Application::~Application()
+{
+	controller->leave();
+	delete controller;
+}
+
+void Application::start() {
+	string cmd, ip;
+	int port, outPort;
+	cout << "command: ";
+	while (cin >> cmd) {
```

Comment:
Consider using a configuration file or command line parser such as:
https://doc.qt.io/qt-5/qcommandlineparser.html
To parse and decide what to do.